### DISCUSSION TITLE
Not accepting my Documents

### DISCUSSION THREAD

**Discussion by TriBlog** [community/community#37768] [GitHub Education]
I am uploading the screenshot every time I get an error that the text is not visible. My screenshot is very clear but it's not accepting. Please help.

**Comment by DSDmark** [comment]
Hey @TriBlog, 

 > Add some more details for better understanding like where you were uploading.

**Reply by TriBlog** [reply]
I was applying got GitHub Student Pack and there I have to upload a document for my verification as a student of the university. I am uploading the screenshot of the Fee submission, but it's not accepting.

**Comment by samus-aran** [comment]
Hi @TriBlog 👋 

Let's see what we can do to assist 🙂 What filesize is your document? Is it in a particular language?

**Reply by TriBlog** [reply]
File Size: 250kb around & English Language.

**Reply by samus-aran** [reply]
Hi @TriBlog thank you for providing more info!
The internal team has confirmed that there was a bug in the uploading of large images and AI reading the text, hence the "text is not eligable". 

It would be awesome if you can try again (using the same file if you are able). This time it should be recognised. Let us know how you get on 👍

**Comment by DSDmark** [comment]
Hi @TriBlog,

as I understand you are facing **file size** problem I guess. There is any error like **File is too small,** if it's the error you should try that:-

I'm not sure if it could help you, but You could try this :- 

Go there [**image size increase**](https://www.imgonline.com.ua/eng/resize-image.php)

Select width and height 1920x1080 and in the bottom there is an **Output image format** section select **PNG-24 (without compression, with transparency support)**  click on **ok** Button your size will be increase.

> I not sure, but minimum size for uploading is 4.5Mb. Don't know why it's not written on the website

whytheway there is whole discussion about that:-

https://github.com/community/community/discussions/36830
